src/skin/qml/qmlskin.cpp

```diff
@@ -109,6 +109,12 @@ QWidget* QmlSkin::loadSkin(QWidget* pParent,
     qmlRegisterType<QmlControlProxy>("Mixxx", 0, 1, "ControlProxy");
 
     QQuickWidget* pWidget = new QQuickWidget(pParent);
+
+    // Enable multisampling for much nicer rendering of shapes/images
+    QSurfaceFormat format;
+    format.setSamples(2);
```

Comment:
...and we should define a constant for easy discoverability of this customization.